modules/mbedtls/create_psa_files.py

```diff
@@ -0,0 +1,89 @@
+#!/bin/python3
+
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+import re
+
+INPUT_FILE="../../../modules/crypto/mbedtls/include/psa/crypto_config.h"
+
+KCONFIG_PATH="Kconfig.psa"
+HEADER_PATH="configs/config-psa.h"
+
+KCONFIG_HEADER="""\
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+# This file was automatically generated by {}.
+# Do not edit it manually.
+
+config PSA_CRYPTO_CLIENT
+	bool
+	help
+	  Promptless symbol to state that there is a PSA crypto API provider
+	  enabled in the system. This allows to select desired PSA_WANT features.
+
+if PSA_CRYPTO_CLIENT
+
+config PSA_CRYPTO_ENABLE_ALL
+	bool
+	prompt "All PSA features enabled"
```

Comment:
```suggestion
	bool "All PSA features"
```

Reply:
All prompts and description for `PSA_CRYPTO_ENABLE_ALL` should be fixed now